README.md

```diff
@@ -377,6 +377,34 @@ okta:
     rateLimit:
       maxRetries: 4
 ```
+
+When you use OAuth 2.0 the full YAML configuration looks like:
+
+```yaml
+okta:
+  client:
+    connectionTimeout: 30 # seconds
+    orgUrl: "https://{yourOktaDomain}" # i.e. https://dev-123456.oktapreview.com
+    proxy:
+      port: null
+      host: null
+      username: null
+      password: null
+    authenticationScheme: "OAUTH2"
+    authorizationMode: "PrivateKey"
+    clientId: "{clientId}"
+    scopes:
+    - scope1
```

Comment:
Can we stick a few real scope examples in here?

Reply:
added real examples.